eng/pipelines/windows.yml

```diff
@@ -107,7 +107,7 @@ jobs:
       buildExtraArguments: /p:RuntimeOS=win10
 
       variables:
-        - nanoQueues: "`(Windows.Nano.1803.Amd64.Open`)windows.10.amd64.serverrs4.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1803-helix-amd64-05227e1-20190509225944"
+        - nanoQueues: "`(Windows.Nano.1809.Amd64.Open`)windows.10.amd64.serverrs5.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1809-helix-amd64-61052b7-20190723211353"
```

Comment:
@MattGal can you please take a look at the failure? It looks to me like the docker image fails to start? (no logs available)

Reply:
Can you tell me which log you think fails to start?  I see  System.Net.Security.Tests fireballed (and has limited logs) due to timeout, everything else just looks like a legit failure.

Reply:
I can't find the one that I was referring to anymore (it was a red error indicating that the log can't be displayed) but still ie here https://mc.dot.net/#/user/dotnet-bot/pr~2Fdotnet~2Fcorefx~2Frefs~2Fpull~2F39734~2Fmerge/test~2Ffunctional~2Fcli~2F~2F/20190726.9/workItem/Microsoft.VisualBasic.Core.Tests/wilogs I can't see the logs for Nano and only get:

```
Executed on a0000GX using docker image mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1809-helix-amd64-61052b7-20190723211353
running %HELIX_CORRELATION_PAYLOAD%\scripts\ba8625e7b6e54d9cbef7bd4bb49f7c6c\execute.cmd in C:\dotnetbuild\work\adb7cdfb-f4c8-417e-a1de-0a7dee34351a\Work\01d733e0-ef1a-49f0-bb65-51154b2c6474\Exec max 900 seconds
```

Reply:
After we stop containers which time out, we take whatever std out the container produces and write them there... this is all it got.  If you have docker and an RS5 machine locally you should be able to reproduce the same.